src/base/datetime/menu/index.ts

```diff
@@ -99,18 +121,39 @@ export class BaseDateTimeMenu extends KucBase {
     `;
   }
 
-  private _handleClickMenu(event: MouseEvent) {
+  updated(_changedProperties: any) {
+    if (!this._firstItemEl) {
+      return;
+    }
+    this._lineHeight = this._firstItemEl.offsetHeight;
+    this._updateScrollOffset();
```

Comment:
@h000780 
>https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html
>When an option is focused that isn't (fully) visible, the listbox's scroll position is updated:

Following the document, the scroll should not called by self of menu.
We need public 3 I/F functions for parrent components(Dropdown) controlling scroll:
1. `scrollToView()` to set focus `_highlightItemEl` when toggling or pressing `arrowup/down` key
2. `scrollToTop()` when pressing `home` key
3. `scrollToBottom()` when pressing `end` key